pkg/service/did/ion.go

```diff
@@ -254,18 +254,16 @@ func (h *ionHandler) GetDID(ctx context.Context, request GetDIDRequest) (*GetDID
 }
 
 // GetDIDs returns all DIDs we have in storage for ION, it is not feasible to get all DIDs from the network
-func (h *ionHandler) GetDIDs(ctx context.Context) (*GetDIDsResponse, error) {
+func (h *ionHandler) GetDIDs(ctx context.Context, deleted bool) (*GetDIDsResponse, error) {
```

Comment:
consider a more meaningful variable name than `deleted`. it is ambiguous whether the presence means you're only getting deleted DIDs, or you're including them. I believe you want the including behavior. in this case `includeDeleted` is a more meaningful name

Reply:
it may be cleaner to keep the code as it was internally. meaning we have a separate `GetDeletedDIDs` method and just the API can interpret the flag.

do need to support all three cases?
- get all DIDs 
- get all DIDs and deleted DIDs
- get just deleted DIDs

I think we should

Reply:
sure happy to support those two